supported_build_configurations.yaml

```diff
@@ -83,3 +83,7 @@ test-pwmin:
 #  - gamepad
 #  - gimbal
 #  - science
+
+test-pololu370-encoder:
```

Comment:
37D

Reply:
I will be looking to rename the whole class I think so i will tackle all of these at once